Makefile

```diff
@@ -308,36 +308,30 @@ create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on AWS
 	fi
 	# Install cert manager.
 	kubectl \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f examples/_out/cert-manager.yaml
 	# Wait for webhook servers to be ready to take requests
 	kubectl \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
 	# Apply provider-components.
 	kubectl \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f examples/_out/provider-components.yaml
 	# Wait for CAPI pod
 	kubectl \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		wait --for=condition=Ready --timeout=5m -n capi-system pod -l control-plane=cluster-api-controller-manager
     # Wait for CAPA pod
 	kubectl \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		wait --for=condition=Ready --timeout=5m -n capa-system pod -l control-plane=capa-controller-manager
 	# Create Cluster.
+	sleep 10
```

Comment:
This adds the sleep back because of frequent failures locally and some in CI:

```
Error from server (InternalError): error when creating "examples/_out/cluster.yaml": Internal error occurred: failed calling webhook "default.cluster.cluster.x-k8s.io": Post https://capi-webhook-service.capi-system.svc:443/mutate-cluster-x-k8s-io-v1alpha3-cluster?timeout=30s: dial tcp 10.101.193.147:443: connect: connection refused
```